Project.toml

```diff
@@ -1,7 +1,19 @@
 name = "AgentEpidemics"
 uuid = "1a1cc00f-f4e8-4f37-91a6-91eca5e02015"
 authors = ["Arindam Saha <arindamsaha1507@gmail.com>"]
-version = "0.1.0"
+version = "0.2.0"
 
 [deps]
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
This section is only for the "proper" dependencies of the package itself.  For example, `Documenter` is only used for building the documentation and is already correctly listed in the `docs` environment, you should add it here.  Likewise, I don't think `Pluto` and `PlutoDevMacros` should be added here.  And `Test` should be moved to the "extra" section:
```toml
[extras]
Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"

[targets]
test = ["Test"]
```
for the test-only packages